pemFioi/conceptViewer-1.0-mobileFirst.js

```diff
@@ -232,20 +260,46 @@ var conceptViewer = {
     this.loadUrl('');
   },
 
+  openInNewWidget: function() {
+    var url = "https://static4.castor-informatique.fr/help/display-documentation.html";
+
+    // we put the language so we can do some operations faster and not depending on the jschannel
+    var x = window.open(url + "?lang=" + window.stringsLanguage);
```

Comment:
You can rename `x` and `c` to better variable names ;)

Reply:
x has been renamed to "fullscreenWindow" and c to "channel"